pkg/byteorder/byteorder.go

```diff
@@ -6,7 +6,7 @@
 //
 //     http://www.apache.org/licenses/LICENSE-2.0
 //
-// Unless required by applicable law or agreed to in writing, software
+// Usless required by applicable law or agreed to in writing, software
```

Comment:
Nit: Change this back to the original correct `Unless` instead of `Usless`

Reply:
Ooops ;-)